language/sygus.tex

```diff
@@ -1581,6 +1569,16 @@ \subsection{Input and Output Logics}
 are planned to be addressed in a future revision of this document
 that includes concrete syntax for function sorts.
 
+\subsubsection{Other Theories}
+
+Many other theories
+are possible beyond those supported in the SMT-LIB standard.
+In this section, we mention theories that are of interest to synthesis applications
+that are not included in the SMT-LIB standard.
+
+\paragraph{Tables}
+TODO
+
```

Comment:
did you mean to include this?

Reply:
It's not critical, but yes, this was a dump of what my local changes were.  I could split this to another PR if you prefer.

Reply:
I don't mind, fine as is